tests/registration/00-simple.t

```diff
@@ -14,119 +14,213 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
 #------------------------------------------------------------------------------
 # Test suite registration
 
+export RND_SUITE_NAME
+export RND_SUITE_SOURCE
+export RND_SUITE_RUNDIR
+export CYLC_RUN_DIR
+
+CYLC_RUN_DIR="$(cylc get-global-config --print-run-dir)"
+
+function make_rnd_suite() {
```

Comment:
I know that at the moment this is the only place this is used, but might these functions be better in test_header?